src/screens/common/create-edit-events/PlanningDetails/PlanningDetails.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useCallback, useMemo } from "react";
+import { View } from "react-native";
+import { useStyles } from "./styles";
+import { PageHeader } from "@app/components/common/PageHeader";
+import { useNavigation } from "@react-navigation/native";
+import Card from "@app/components/ui/Card";
+import { useForm } from "react-hook-form";
+import { FormInput, FormTabSelect } from "@app/components/forms";
+import { acceptanceType } from "@app/utils/constants";
+import Button from "@app/components/ui/Button";
+import { getErrorMessage } from "@app/components/forms";
+import { regexValidator } from "@app/utils/validators";
+import { PlanningDetailsReduxProps } from "./container";
+
+type PlanningDetailsProps = PlanningDetailsReduxProps;
+
+const PlanningDetails: React.FC<PlanningDetailsProps> = ({
+  createData,
+  updateCreateData,
+}) => {
+  const styles = useStyles();
+  const navigation = useNavigation();
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const acceptanceOptions = useMemo(
```

Comment:
You can define this as a constant outside the component so that useMemo is not needed.